cfpb/sbl-frontend
# deployment(docker/nginx): addressing the 403's -- fonts and the favicon

closes #770 

## Changes
- deployment(nginx): added routes to open fonts and the favicon to routes / and /static/assets/


## How to Test
- Ensure it builds with `yarn docker-build-and-run`
- Test in the dev or staging, ensure the fonts and favicons no longer get 403's or 404's
